backend/api/chat_orchestrator.py

```
@@ -65,11 +67,12 @@ class ChatOrchestrator:
     Enhanced with optional LangChain integration for improved AI interactions
     """
     
-    def __init__(self, scenario_data: Dict[str, Any], enable_langchain: bool = True):
+    def __init__(self, scenario_data: Dict[str, Any], enable_langchain: bool = True, is_professor_test: bool = False):
```

Ruff (0.14.1)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)